src/js/stores/BallotStore.js

```diff
@@ -77,7 +77,8 @@ class BallotStore extends FluxMapStore {
 
   reduce (state, action) {
 
-    if (action.res.success === false)
+    if (typeof action.res === "undefined" ||
```

Comment:
I would suggest just getting rid of this line. I can't think of any time when the action.res.success is false and we need to shortcut. The returning of state is covered by the default case.